db/seeds.rb

```diff
@@ -15,6 +15,10 @@
 
 puts "Previous seeds have been deleted!"
 
+puts "Deleting previous 1000 cloudinary avatars..."
+Cloudinary::Api.delete_all_resources
+puts "Previous 1000 cloudinary avatars deleted!"
+
```

Comment:
I could not find any other better way of deleting the avatars. Can look into this further in the future if needed.